crates/telio-nurse/src/nurse.rs

```diff
@@ -267,24 +264,22 @@ impl State {
     }
 
     fn meshnet_id() -> Uuid {
-        lana!(fetch_context)
-            .map(|context| context.application.config.current_state.internal_meshnet.fp)
-            .unwrap_or(None)
-            .map(|fp| {
-                Uuid::parse_str(&fp).unwrap_or_else(|_| {
-                    telio_log_error!(
-                        "Failed to parse moose meshnet id ({:?}), generating a new one",
-                        fp
-                    );
-                    Uuid::new_v4()
-                })
-            })
-            .unwrap_or_else(|| {
-                telio_log_trace!(
-                    "Could not find stored meshnet id by moose, generating a fresh one"
+        telio_lana::fetch_context_string(String::from(
+            "contest.application.libtelioapp.config.internal_meshnet.fp",
```

Comment:
is the new moose tracker working by providing string keys for it? I find it pretty fragile as there's no type safety, we can easily produce libtelio binaries with incorrect strings in here :/

Reply:
As I actually did "contest" :))
The way it works is that you call fetch_context_string and you give it a path and if the path is valid and there is a string, it will return it to you.

Reply:
Yes, I think it would be nice. But another update, would break again the compatibility with all the integrations of moose. So I don't think the guys will enjoy to do that.